app/views/layouts/_footer.html.erb

```diff
@@ -19,7 +19,7 @@
           <li><a href="/mentorship">Software Mentorship</a></li>
           <li><a href="/scholarships">Scholarship Program</a></li>
           <li><a href="/employers">Employer Services</a></li>
-          <li><a href="/deploy">Web Application Development</a></li>
+          <li><a href="http://deployopcode.io/">Web Application Development</a></li>
```

Comment:
👍 

I thought about.  I just didn't know if you minded the link saying `http://operationcode.org/deploy`.